### ISSUE TITLE
Build A-Frame skeleton for Scenario 3

### ISSUE THREAD

**Issue by jasonalderman-valtech** [valtech-sd/spyglass#62]
Blocked by #52

AC:

- [x] Position a tab panel to side of bottle
- [x] Render icons for tab labels and indicate selected tab
- [x] Render text and image content for contents of each tab
- [x] Content positioned based on marker position (in space)
- ~~[ ]~~ ~~Interaction hinting images added to scenario 3 (and 2)~~ MOVED TO NEW ISSUE
